client/my-sites/domains/domain-management/settings/cards/domain-redirect-card.tsx

```diff
@@ -43,6 +41,7 @@ export default function DomainRedirectCard( {
 	const [ protocol, setProtocol ] = useState( 'https' );
 	const [ isValidUrl, setIsValidUrl ] = useState( true );
 	const [ errorMessage, setErrorMessage ] = useState( '' );
+	const pointsToWpcom = domain?.pointsToWpcom ?? false;
```

Comment:
```suggestion
	const pointsToWpcom = domain.pointsToWpcom;
```

This field isn't optional.

Reply:
Updated on https://github.com/Automattic/wp-calypso/pull/80716/commits/6793928621bade5175ea11704898e874c7bef9cd